modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,45 @@
+# Import needed libraries
+from PIL import Image  # image libraries and operations
+import os, sys  # directory libraries
+import imghdr  # image file recognition
+import numpy  # numpy arrays
+
+# Sample Paths
+# /mnt/c/Users/Owner/Pictures/SamplePictures/
+# /mnt/c/Users/Tayo/Documents/SE2CS4365/TestImages/
+
+# list of image types
+imgfiles = ['rgb', 'gif', 'pbm', 'pgm', 'ppm', 'tiff', 'rast', 'xbm', 'jpeg', 'jpg', 'bmp', 'png', 'webp', 'exr']
+
+
+# Splitting up this module into smaller ones
+
+# Given a filepath and size, resizeFolder will enter the folder
+# and edit the files to be square, with lxw as sizexsize
+def resizeFolder(path, size):
```

Comment:
Yeah I was thinking that too, but there are a lot of different transforms that Pillow has, and a lot of them could be used to make application of ML algorithms easier. To make a module for each and every one of them would be a lot, that's why I decided to focus on just resizing, because this will almost always be used